docs/networking.md

```diff
@@ -0,0 +1,175 @@
+# How to use networking
+
+cloud-hypervisor can emulate one or more virtual network interfaces, represented at the hypervisor host by [tap devices](https://www.kernel.org/doc/Documentation/networking/tuntap.txt"). This guide briefly describes how to setup and use networking with cloud-hypevisor.
+
+## Start cloud-hypervisor with net devices
+
+Use one or more `--net` command-line argument from __cloud-hypervisor__ to specify the emulation of one or more virtual NIC's. The example below instructs cloud-hypervisor to emulate for instance 2 virtual NIC's:
+
+```bash
+./cloud-hypervisor \
+    --cpus 4 \
+    --memory "size=512,file=/dev/shm" \
```

Comment:
I think you want `size=512M...` here.

Reply:
And you don't need the `file=/dev/shm` part.

Reply:
sure. will change this.